docs/team-process.md

```diff
@@ -70,13 +70,15 @@ If dependencies have no patch, replace it with an alternative, or wait for the l
 
     NOTE: If any npm packages are updated, update the node cache version in the circle config. You can do this by searching within `config.yml` for vX-npm and vX-cypress where X is the current version of the cache key, then increment the version found.
 
-3. `terraform`: check for a newer version on the [Terraform site](https://www.terraform.io/downloads).
+4. `terraform`: check for a newer version on the [Terraform site](https://www.terraform.io/downloads).
 
     - Once verification is complete, you will need to increment the docker image version being used in `.circleci/config.yml` and publish a docker image tagged with the incremented version number to ECR for both Flexion and USTC accounts.
 
-4. `docker`: Update [docker base image](https://hub.docker.com/r/cypress/base/tags?page=1&name=14.) if an update is available for the current node version the project is using.
+5. `docker`: Update [docker base image](https://hub.docker.com/r/cypress/base/tags?page=1&name=14.) if an update is available for the current node version the project is using.
 
-5. (Optional) Check through the list of caveats to see if any of the documented issues have been resolved.
+    See [here](ci-cd.md#docker) for the documentation to create and push the updated docker container for use in CircleCI.
+
+6. (Optional) Check through the list of caveats to see if any of the documented issues have been resolved.
 
```

Comment:
I didn't see in the exp runs if y'all did a CircleCI build with a forced migration. Am I missing that? Seeing that it's way below this section in `Validating Updates` I know it's not too apparent that it's part of the update steps.

Reply:
Ooooh, good catch.  We didn't do that.